.github/workflows/check-links.yml

```
@@ -31,6 +31,13 @@ jobs:
         # 429: Too many requests
         args: >
           --accept 429
+          --exclude "*/n$"
+          --exclude "*/n#$"
```

Use regex, not glob pattern.
```suggestion
          --exclude ".*/n$"
          --exclude ".*/n#$"
```